server/planning/events/events_spike.py

```diff
@@ -62,6 +62,50 @@ def update(self, id, updates, original):
 
         return item
 
+    def on_updated(self, updates, original):
+        # Spike associated planning
+        planning_spike_service = get_resource_service('planning_spike')
+        query = {
+            'query': {'bool': {'must': {'term': {'event_item': str(original[config.ID_FIELD])}}}}
+        }
+        results = get_resource_service('planning').search(query)
+        spiked_items = []
+        if len(results.docs) > 0:
+            for planning in results.docs:
+                if planning['state'] == WORKFLOW_STATE.DRAFT:
+                    spiked_items.append(planning_spike_service.patch(planning[config.ID_FIELD],
+                                                                     {'state': 'spiked'}))
+
+            # When a planning item associated with this event is spiked
+            # If there were any failures in removing assignments
+            # Send those notifications here
+
+            assignments_to_fetch = []
+            assignments_service = get_resource_service('assignments')
+            for planning in spiked_items:
```

Comment:
here we can query by planning id using elasticsearch as `assignments` store planning id.